sdk/src/main/java/io/dapr/client/DaprClientGrpc.java

```diff
@@ -629,18 +630,14 @@ public void start(final Listener<RespT> responseListener, final Metadata headers
    */
   private static SpanContext extractOpenCensusSpanContext(Context openTelemetryContext) {
     Map<String, String> map = new HashMap<>();
-
-    OpenTelemetry.getPropagators().getHttpTextFormat().inject(
-        openTelemetryContext,
-        map,
-        new HttpTextFormat.Setter<Map<String, String>>() {
-          @Override
-          public void set(Map<String, String> map, String key, String value) {
-            if (map != null) {
-              map.put(key, value);
-            }
+    TextMapPropagator.Setter<Map<String, String>> setter =
```

Comment:
Please, move this to a static final field. I know the old code was not like that but this would be an enhancement over creating a new setter on every call.